src/TsTypes.ts

```diff
@@ -1,5 +1,9 @@
 import { camelCase, upperFirst } from 'lodash'
 
+const multiLineCommentStart =  '/** '
+const multiLineCommentIndent = '    '
```

Comment:
I am using vim with https://github.com/Quramy/tsuquyomi and some other typescript plugins to get similar code completion, hints, jump to definition/references, etc..  Under the covers, the plugin uses a service from `tsc` and I guess VSCode uses this same service.

I found the hinting works when the interface is in the same file. But if it is imported, the hint points to the import. I have to jump to the definition to see the comment.  Perhaps this is the difference in @donaldpipowitch and @bcherny test?  Maybe one day `tsc` will be enhanced to provide hints to interfaces that are external to the current file?

@bcherny - what do you think? Should I retire the code that does the single line description?  I agree with @donaldpipowitch that it is really useful to have this info available to editors.